server/model/model.py

```diff
@@ -42,7 +42,7 @@ class EventTypes(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     user_id = db.Column(db.Integer, db.ForeignKey('users.id'))
     title = db.Column(db.String(64), nullable=False)
-    url = db.Column(db.String(64), nullable=False)
+    url = db.Column(db.String(64), nullable=False, unique=True)
```

Comment:
Added unique constraint to event type url, since we need that to figure out the event type

Reply:
good point 👍